jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/CombinedResource.java

```diff
@@ -301,19 +295,7 @@ public void copyTo(Path destination) throws IOException
             else
             {
                 ensureDirExists(pathTo.getParent());
-                Path pathFrom = r.getPath();
-                if (pathFrom != null)
-                {
-                    Files.copy(pathFrom, pathTo, StandardCopyOption.COPY_ATTRIBUTES, StandardCopyOption.REPLACE_EXISTING);
-                }
-                else
-                {
-                    // use old school stream based copy
-                    try (InputStream in = r.newInputStream(); OutputStream out = Files.newOutputStream(pathTo))
-                    {
-                        IO.copy(in, out);
-                    }
-                }
+                r.copyTo(pathTo);
```

Comment:
It feels like the behavior of `copyTo()` about what it does when the target exists or not (i.e.: the `Files.copy()` flags) should be javadoc'ed.